applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -315,7 +315,7 @@ upload_csv(Context) ->
     Now = erlang:now(),
     {'ok', {Count, Rates}} = process_upload_file(Context),
     lager:debug("trying to save ~b rates (took ~b ms to process)", [Count, wh_util:elapsed_ms(Now)]),
-    _  = crossbar_doc:save(cb_context:set_doc(Context, Rates), [{'publish_doc', 'false'}]),
+    lists:foreach(fun(Rate) -> crossbar_doc:ensure_saved(cb_context:set_doc(Context, Rate), [{'publish_doc', 'false'}]) end, Rates),
```

Comment:
This is related to the dev posting at https://groups.google.com/forum/#!topic/2600hz-dev/RVa2jV_kTX8 in which Lucas said that crossbar_doc:save would not update existing rates, but crossbar_doc:ensure_saved would.